src/htable.cpp

```diff
@@ -1031,16 +1018,8 @@ void HeadedTable::setTreeMode(bool tm)
 // update table Head !!
 void HeadedTable::setSortedCol(int col)
 {
-    {
-        int old_sorted = sorted_col;
-        sorted_col = col;
-        //		printf("old_sorted=%d\n",old_sorted);
-        //		printf("sorted_col=%d\n",sorted_col);
-        if (old_sorted != -1 && old_sorted < ncols) // ncols bug
-            updateHeading(old_sorted);
-        if (col != -1 && col < ncols)
-            updateHeading(col);
-    }
+    int old_sorted = sorted_col;
```

Comment:
old_sorted seems to be unused now

Reply:
Like below: "Not important because there's a long long way to a clean code." This is only the first attempt.